bin/stashcp

```diff
@@ -97,6 +97,14 @@ def doStashCpSingle(sourceFile, destination, cache, debug=False):
     payload['start2']=start2
     payload['end2']=end2
     
+    with open(".job.ad") as job_file:
+        for line in job_file.readLines():
+            if "AutoClusterAttrs" in line:
```

Comment:
Alright, I think I have something that may be of use
```python
        for line in job_file.readLines():
            match = re.match('Owner|ProjectName', line, re.IGNORECASE)
            if match:
                payload[match.group()] = line
```
re.match only matches at the beginning of the string. Looking good?